src/dapps/BuySubdomain/BuySubdomain.vue

```diff
@@ -119,7 +122,12 @@ export default {
   },
   methods: {
     debounceInput: web3.utils._.debounce(function(e) {
-      this.domainName = normalise(e.target.value);
+      if (e.target.value.indexOf(' ') >= 0) {
```

Comment:
it cannot also have `.` and other invalid characters also `normalise` can fail, may be put normalise inside try catch?

Reply:
changed 👍